pkg/security/tests/resolvconf/resolvconf.go

```diff
@@ -0,0 +1,82 @@
+// Extracted from /moby/libnetwork/resolvconf/resolvconf.go
```

Comment:
could you plain why it is needed to copy code from moby project? is it because this code has been removed from the docker repo version that we are now vendoring?

Reply:
It was because `libnetwork` standalone repo is deprecated and was merged in `moby/moby`, but only starting newest version of Docker, which I did not want to migrate to but had no choice to in the end, so not necessary anymore.

Removed.